kt_jvm_grpc.bzl

```diff
@@ -160,6 +160,74 @@ def kt_jvm_grpc_library(
         features = features,
     )
 
+def kt_jvm_grpc_no_java_library(
```

Comment:
I agree that it is weird. Here are some alternate suggestions: `kt_jvm_grpc_basic_library`, `kt_jvm_grpc_simple_library`, `kt_jvm_grpc_barebones_library`; also all of the ideas with the new term in between `jvm` and `grpc` instead.

I suspect it will be difficult to find a good name as we're trying to distinguish ourselves from another rule type which we cannot rename. I'm not set on a particular naming scheme but want to make sure it expresses the functional difference with the sibling rule type as clearly as possible.

Reply:
... I like those suggestions even less than the "no_java" version.

🤷🏻  What to do when the only sensible name is already blocked.

So I think you should just use the one you had.